EIPS/eip-6672.md

```diff
@@ -0,0 +1,247 @@
+---
+eip: 6672
+title: Multi-redeemable NFTs
+description: An extension of ERC-721 which enables an NFT to be redeemed in multiple scenarios for either a physical or digital object
+author: RE:DREAMER Lab <dev@redreamer.io>, Archie Chang (@ArchieR7) <archie@redreamer.io>, Kai Yu (@chihkaiyu) <kai@redreamer.io>, Yonathan Randyanto (@Randyanto) <randy@redreamer.io>
+discussions-to: https://ethereum-magicians.org/t/eip-6672-multi-redeemable-nfts/13276
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-02-21
+requires: 165, 721
+---
+
+## Abstract
+
+This EIP proposes an extension to the [ERC-721](./eip-721.md) standard for Non-Fungible Tokens (NFTs) to enable multi-redeemable NFTs. This extension would allow an NFT to be redeemed in multiple scenarios for either physical or digital objects and maintain a record of its redemption status on the blockchain.
+
+## Motivation
+
+[ERC-5560](./eip-5560.md) enables only one-time redemption of an NFT, which means the same NFT cannot be re-used for another redemption from different campaigns or events. Our proposed multi-redeemable NFT is an improved alternative to the [ERC-5560](./eip-5560.md) redeemable NFT.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+**Every contract compliant with this EIP MUST implement `ERC-6672` and `ERC-721` interfaces.**
```

Comment:
Because these are in backticks, you cane use the interface name directly:

```suggestion
**Every contract compliant with this EIP MUST implement `ERC6672` and `ERC721` interfaces.**
```

Reply:
changed